ui/main/src/app/services/notifications/NotificationDecision.ts

```diff
@@ -150,4 +152,11 @@ export class NotificationDecision {
             this.cleanSentCards();
         }, this.CLEAN_CARDS_PERIOD);
     }
+
+    private static shouldCardBeNotified(card: Card): boolean {
+        return (
+            !card.actions?.includes(CardAction.NOT_NOTIFIED) ||
+            UsersService.getCurrentUserWithPerimeters().userData.entities.includes(card.publisher)
```

Comment:
i change my mind : just test if NOT NOTIFIED

Reply:
Done.